src/PKSim.Infrastructure/ORM/Repositories/IndividualParametersSameFormulaOrValueForAllSpeciesRepository.cs

```diff
@@ -42,14 +42,21 @@ protected override void DoStart()
          foreach (var individualParameter in flatIndividualParametersSameFormulaOrValue)
          {
             var (containerId, parameterName, isSameFormula, _, _) = individualParameter;
+
+            //For ontogeny table parameter, we NEVER use the same formula as they are clearly individual specific parameters but are defined as constant in the database
+            if (parameterName.IsOneOf(ONTOGENY_FACTOR_TABLE, ONTOGENY_FACTOR_GI_TABLE, ONTOGENY_FACTOR_ALBUMIN_TABLE, ONTOGENY_FACTOR_AGP_TABLE))
```

Comment:
@Yuri05 I am not sure if this is the best way to exclude them, But they are returned as the same because the value in DB is 1 but this is clearly BS

Reply:
OK for me.
Technically those parameters are the same, so we need to exclude them explicitly somewhere.